nixos/modules/services/networking/jormungandr.nix

```diff
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.jormungandr;
+
+  inherit (lib) mkEnableOption mkIf mkOption;
+  inherit (lib) optionalString types;
+
+  dataDir = "/var/lib/jormungandr";
+
+  # Default settings so far, as the service matures we will
+  # move these out as separate settings
+  configSettings = {
```

Comment:
Are you familiar with [rfc42](https://github.com/NixOS/rfcs/pull/42) yet? Specifically the `settings` option may be of interest to you.

Reply:
I am not, let me review it.

Reply:
This is pretty nice, I will be maybe use it later as the upstream product matures as right now this is not needed. Right now I'm doing this like this because it requires a valid JSON and putting it as it is messed my indentation.